src/sentry/conf/server.py

```diff
@@ -1892,8 +1892,6 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
     "organizations:spike-protection-decay-heuristic": False,
     # Enable Slack messages using Block Kit
     "organizations:slack-block-kit": False,
-    # Enable new Slack message formatting
-    "organizations:slack-formatting-update": False,
```

Comment:
We've decided to release the block kit refactor along with the formatting changes, so I'm consolidating the flags.